data/json/obsoletion/obsolete_mapgen.json

```diff
@@ -113,109 +113,102 @@
     }
   },
   {
+    "type": "mapgen",
     "method": "json",
+    "om_terrain": [ [ "campground_1a_npc", "campground_1b_npc" ], [ "campground_2a_npc", "campground_2b_npc" ] ],
     "object": {
-      "items": {
```

Comment:
I'm not sure if it's new obsoletions or modified old ones - but if possible consider moving them to a new file so that the old obsoletions from 0.G clear out in https://github.com/CleverRaven/Cataclysm-DDA/pull/64399

Reply:
This is from the previous cycle, like everything else in obsolete_mapgen. But sure, I can move it to a new file.